modular_skyrat/modules/station_traits/code/positive.dm

```diff
@@ -0,0 +1,8 @@
+/datum/station_trait/sentient_announcer
+	name = "Sentient Announcer"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 10
```

Comment:
drop this to STATION_TRAIT_NEGATIVE and lower it's weight, this is exceedingly common rn if the weight code works the way I think it does.

Reply:
the initial implementation of this was pretty quick, so yeah, from testing its too common, and would be better suited as a negative trait